packages/host/app/templates/card.gts

```diff
@@ -68,7 +69,9 @@ class HostModeComponent extends Component<HostModeComponentSignature> {
     return false;
   }
 
-  private get cardContext(): HostModeCardContext {
+  @provide(CardContextName)
+  // @ts-ignore "context" is declared but not used
+  private get context(): HostModeCardContext {
```

Comment:
im not sure if we possibly want to provide another context name here like HostCardContextName since HostCardContext by type seems to exclude prerenderedSearch but I dont know why

Reply:
I like to err on the side of adding more context names as opposed to overloading the default context.